build.sbt

```diff
@@ -36,7 +35,7 @@ lazy val core = (project in file("acdc-core")).
     libraryDependencies ++= Seq(
       "com.typesafe.slick" %% "slick" % slickVersion,
       "com.typesafe.slick" %% "slick-hikaricp" % slickVersion,
-      "org.postgresql" % "postgresql" % "42.4.+"
+      "org.postgresql" % "postgresql" % "42.5.0"
```

Comment:
@realstraw maybe `42.5.+`?

Reply:
i think fixed version makes more sense to be consistent with the rest, we are also not publishing this as library neither